test/dynamo/test_functions.py

```diff
@@ -1723,8 +1723,14 @@ def pow2_kernel(
     @requires_cuda()
     @requires_triton()
     @common_utils.parametrize("grad", [False, True])
-    @common_utils.parametrize("backend", ["eager", "aot_eager"])
+    @common_utils.parametrize("backend", ["eager", "aot_eager", "inductor"])
+    @patch.object(torch._inductor.config, "implicit_fallbacks", False)
     def test_triton_kernel_native(self, grad, backend):
+        if backend == "inductor" and grad is True:
```

Comment:
@zou3519 @bdhirsh

This is surprising. It fails with
```
RuntimeError: a leaf Variable that requires grad is being used in an in-place operation.
```
in P857566445.

Looking at it with a debugger, we are calling copy_ on a faketensor that requires grad. Looks like aot_eager does not execute this codepath.